doc/htmldoc/developer_space/index.rst

```diff
@@ -0,0 +1,53 @@
+.. _developer_space:
+
+Developer space
+===============
+
+Here is all documentation pertaining to the development of NEST.
+It is documentation for anyone needing to touch the code or documentation (also includes SLI, since it is more complex
+and likely used by developers or very advanced users).
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Install from source
+
+   ../installation/linux_install
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Workflows
+   :glob:
+
+   workflows/*
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Guidelines
+   :glob:
+
+   guidelines/*
+   guidelines/styleguide/styleguide
+   guidelines/styleguide/vim_support_sli
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Templates
+   :glob:
+
+   templates/*
+
+.. toctree::
+   :maxdepth: 1
+   :caption: SLI docs
+   :glob:
+
+   sli_docs/*
```

Comment:
This shows all the links under `sli_doc` and also a link to that page. Should the `*` be removed here?

Reply:
fixed